src/background.js

```diff
@@ -1,5 +1,5 @@
 import urlRegex from 'url-regex'
-
+import 'src/util/url-filter.js'
```

Comment:
no need to import this in the background entrypoint either. I'll import this in the appropriate place later to ensure all URLs that go through the extension get normalized by this function.